Signal/src/ViewControllers/AddToBlockListViewController.m

```diff
@@ -0,0 +1,346 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "AddToBlockListViewController.h"
+#import "CountryCodeViewController.h"
+#import "PhoneNumber.h"
+#import "StringUtil.h"
+#import "UIFont+OWS.h"
+#import "UIUtil.h"
+#import "UIView+OWS.h"
+#import "ViewControllerUtils.h"
+#import <SignalServiceKit/PhoneNumberUtil.h>
+#import <SignalServiceKit/TSBlockingManager.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSString * const kAddToBlockListViewControllerCellIdentifier = @"kAddToBlockListViewControllerCellIdentifier";
+
+#pragma mark -
+
+// TODO: Add a list of contacts to make it easier to block contacts.
+@interface AddToBlockListViewController () <CountryCodeViewControllerDelegate, UITextFieldDelegate>
+
+@property (nonatomic, readonly) TSBlockingManager *blockingManager;
+
+@property (nonatomic) UIButton *countryNameButton;
+@property (nonatomic) UIButton *countryCodeButton;
+
+@property (nonatomic) UITextField *phoneNumberTextField;
+
+@property (nonatomic) UIButton *blockButton;
+
+@property (nonatomic) NSString *callingCode;
+
+@end
+
+#pragma mark -
+
+@implementation AddToBlockListViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    [self.navigationController.navigationBar setTranslucent:NO];
```

Comment:
I'm just trying to be consistent with the other VCs in our project, all (?) of which seem to do this.  I'm not clear on why this wasn't done with UIAppearance, but it seemed orthogonal to this work.